AppController/djinn.rb

```diff
@@ -5029,6 +5034,9 @@ def scale_up_instances(needed_appservers)
         @last_scaling_time = Time.now.to_i
         Djinn.log_info("Added the following nodes: #{roles_needed}.")
       }
+
+      # Make sure we have the proper list of blobservers configured.
+      update_blob_servers
```

Comment:
I don't think we can assume that the blob server is up and running on the new instance at this point.